### DISCUSSION TITLE
Plot specific motif enrichment heatmap / footprints

### DISCUSSION THREAD

**Discussion by boconnell89** [GreenleafLab/ArchR#1511] [Questions / Documentation]
Is there a way to add a specific motif (e.g. CTCF) to the motif enrichment **or** footprint plots?  I'm trying to use a couple of motifs as controls between experiments.

**Comment by rcorces** [comment]
I'm not sure exactly what you mean.
In `plotFootprints()` there is a `name` parameter to do this - https://www.archrproject.com/reference/plotFootprints.html

For motif enrichment, the examples in the manual label the TFs using ggrepel which could be forced to label whatever you want to label. If you're talking about the motif enrichment heatmap after running `peakAnnoEnrichment()`, then there isnt a great way to do this specifically. Generally speaking, the plotting functions in ArchR are meant to be helpful but not comprehensive. So there may be times where you need to take the `SummarizedExperiment` that ArchR gives you and write your own code to plot exactly what you want.
